argocd/schema_application_set.go

```diff
@@ -276,6 +276,11 @@ func applicationSetGitGeneratorSchemaV0() *schema.Schema {
 						},
 					},
 				},
+				"path_param_prefix": {
+					Type:        schema.TypeString,
+					Description: "Prefix for all path-related parameter names. Only applicable when using the `matrix` generator. See https://argo-cd.readthedocs.io/en/stable/operator-manual/applicationset/Generators-Matrix/#example-two-git-generators-using-pathparamprefix",
```

Comment:
The prefix is only necessary when using a matrix with two git generators, but it looks like it's supported whenever a git generator is used, even on it's own.

Reply:
Ah, my bad! I see there is also mention of it in the [Git Generator docs](https://argo-cd.readthedocs.io/en/stable/operator-manual/applicationset/Generators-Git/). Have reverted that change - thanks!